dart/lib/src/enricher/io_enricher_event_processor.dart

```diff
@@ -70,10 +70,12 @@ class IoEnricherEventProcessor extends EventProcessor {
         // This throws sometimes for some reason
         // https://github.com/flutter/flutter/issues/83921
         executable = Platform.executable;
-      } catch (exception) {
+      } catch (exception, stackTrace) {
         _options.logger(
           SentryLevel.info,
```

Comment:
lets change to `error` then?

Reply:
I meant instead of `SentryLevel.info`, it should be `SentryLevel.error`, so its not resolved uet